plugins/groups/src/Router.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { rootRouteRef, viewGroupRouteRef, viewMemberRouteRef } from './plugin';
+import { GroupsList, GroupView, MemberView } from './components';
+
+export const Router = () => (
+  <Routes>
+    <Route path={`/${rootRouteRef.path}`} element={<GroupsList />} />
+    <Route path={`/${viewGroupRouteRef.path}`} element={<GroupView />} />
+    <Route path={`/${viewMemberRouteRef.path}`} element={<MemberView />} />
```

Comment:
I would expect Groups and Users to be part of the entitiy page (and the routes there). We just did that for APIs #2987. That allows to automatically link to the group and user pages from search and let everyone customize these pages in the app.